app/javascript/packages/document-capture/components/acuant-selfie-capture-canvas.scss

```diff
@@ -0,0 +1,4 @@
+.document-capture-selfie-feedback {
+  color: $white;
```

Comment:
@aduth From a LGDS perspective, is it better to use `$white`/`$black` or `color('white')`/`color('black')` in this file? 

 There is some [variation](https://github.com/18F/identity-idp/blob/6e3a7c0bc3b02179c4f98c0b52b60601f53d7b36/app/javascript/packages/document-capture/components/select-error.scss#L2) in how colors are managed in the `document-capture/components/…` stylesheets, so I wanted to check with you.  (Also curious to hear from other devs!)

Reply:
I wasn't aware that `$white` still existed as a variable. I'd generally prefer going through the `color( ... )` utility function.

Reply:
> I wasn't aware that `$white` still existed as a variable.

Based on the failing build, it may not 😄 

```
  sassMessage: 'Undefined variable.',
...
  span: {
    text: '$white',
```

I think the `$white` variable only exists in the context of the email stylesheet, though it'd probably be good to remove it from there as well for consistency.

Reply:
>Based on the failing build, it may not 😄

Hahaha, very true!  I fixed this to use the `color(...)` utility function.